docs/guides/cloud/deploy/render.rst

```diff
@@ -0,0 +1,27 @@
+.. _ref_guide_cloud_deploy_render:
+
+======
+Render
+======
+
+:edb-alt-title: Deploying applications built on EdgeDB Cloud to Render
+
+1. Push project to GitHub or some other Git remote repository
+2. Create and make note of a secret key for your EdgeDB Cloud instance
+3. From Render's dashboard, click "New > Web Service"
+4. Import your project's repository
+5. In your new project's page, click "Environment" to add two variables:
```

Comment:
These can be set directly from the import without having to go through the project page. That's probably the most straightforward way for the purposes of the guide.
![CleanShot 2024-03-06 at 07 45 27@2x](https://github.com/edgedb/edgedb/assets/2027711/7241f7b0-c3d5-48ce-ac07-fa5fb509fea3)